Source/WebKit/UIProcess/API/Swift/WebPage+Download.swift

```diff
@@ -25,8 +25,9 @@
 
 import Foundation
 
-extension WebPage_v0 {
+extension WebPage {
     @MainActor
+    @_spi(Private)
     public struct Download: Sendable {
```

Comment:
Why is this `@MainActor` and `Sendable`?

Reply:
ah true, I don't think it needs the @MainActor . Thanks!!

Reply:
I think there were a couple of other types with both annotations, might be worth a quick audit. In general, if a type is bound to a global actor it is also Sendable, because the concurrency checker knows you won’t access it improperly. But writing both is a bit confusing w.r.t. intent. I think having a type that’s sendable but with one or two `@MainActor` vars or funds could make sense, but is probably rare.